app/_assets/stylesheets/footer.less

```
@@ -11,38 +11,25 @@
    MARKETING -> FOOTER
    ==================================================== */
 
-// Animations
-@keyframes mashape-ape-animation {
-  0% {
-    background-position: 0% 0%;
-  }
-  100% {
-    background-position: 200% 0%;
-  }
-}
-@anim--speed: 0.3s;
-
 // Subscribe Form
 .footer-form-container {
   display: flex;
   flex-direction: row;
 
   #subscribe-input {
-    width: calc(100% - 200px);
+    width: 70%;
     height: 40px;
-    padding-left: 10px;
     border: 1px solid #ccc;
-    display: inline-block;
     border-radius: 5px;
     font-size: 16px;
   }
 
   #footer-form-button {
-    width: 22%;
+    width: 25%;
     font-size: 16px;
     font-weight: normal;
```

```suggestion
    font-weight: 500;
```
To match the .com site